neurons/_miner/miner_session.py

```
@@ -3,39 +3,40 @@
 import os
 import time
 import traceback
-from typing import Tuple, Union
 
 import bittensor as bt
 import websocket
+from bittensor.core.extrinsics.serving import serve_extrinsic
+from fastapi.exceptions import RequestValidationError
```

Flake8 (7.3.0)

[error] 'fastapi.exceptions.RequestValidationError' imported but unused

(F401)